doc/cephadm/services/osd.rst

```diff
@@ -518,19 +523,19 @@ Create a file called (for example) ``osd_spec.yml``:
 
 This means :
 
-#. Turn any available device (ceph-volume decides what 'available' is) into an
-   OSD on all hosts that match the glob pattern '*'. (The glob pattern matches
-   against the registered hosts from `host ls`) A more detailed section on
-   host_pattern is available below.
+#. Turn any available device (``ceph-volume`` decides what ``available`` means)
+   into an OSD on all hosts that match the glob pattern '``*``'. (The glob
+   pattern matches against the registered hosts returned by the command ``host
+   ls``) A more detailed section on ``host_pattern`` is available below.
```

Comment:
A bit of a nitpick since it was the same before your edits, but maybe we want to write out the full `ceph orch host ls` command to avoid potential confusion.

Reply:
Yeah, let's do it.

Reply:
...and now it's done.